src/agent/command_handler/src/command_handler.cpp

```diff
@@ -2,10 +2,22 @@
 
 namespace command_handler
 {
-    const std::unordered_map<std::string, std::pair<std::string, module_command::CommandExecutionMode>>
-        VALID_COMMANDS_MAP = {
-            {"set-group", {"CentralizedConfiguration", module_command::CommandExecutionMode::SYNC}},
-            {"update-group", {"CentralizedConfiguration", module_command::CommandExecutionMode::SYNC}}};
+    struct CommandDetails
+    {
+        std::string module;
+        module_command::CommandExecutionMode executionMode;
+        bool requiresArguments;
```

Comment:
We can remove this if we check the arguments vector size.

Reply:
Done!